test/javascripts/unit/admin/modules/deselect_taxon_click_test.js

```diff
@@ -0,0 +1,92 @@
+
```

Comment:
Stray empty lines at start of file.

Reply:
I've removed these in 09d71d7bd2a2338af6902367d70c65fad4b4c95